includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -86,6 +129,7 @@ public function get_settings() {
 	 */
 	public function update_settings( WP_REST_Request $request ) {
 		$this->update_is_payment_request_enabled( $request );
+		$this->update_payment_request_settings( $request );
```

Comment:
One thing I noticed is that the fetch call to this endpoint will catch exceptions but there aren't any exceptions thrown in this class. Are there conceivable paths where we'd want to throw an exception(an update fails?) or are we mainly just catching network errors?

Reply:
The `POST` request to `update_settings` could fail when the validation on the input fails (through `validate_callback`) - is that the endpoint you're referring to?
In that case, an exception can be thrown by the validators.

Reply:
Sounds good 👍